src/status_im/contexts/communities/events.cljs

```diff
@@ -135,21 +123,28 @@
                 {}
                 categories))}))
 
+(rf/reg-event-fx :community/fetch-low-priority
+ (fn []
+   {:fx [[:json-rpc/call
+          [{:method      "wakuext_checkAndDeletePendingRequestToJoinCommunity"
+            :params      []
+            :js-response true
+            :on-success  [:sanitize-messages-and-process-response]
+            :on-error    #(log/info "failed to fetch communities" %)}
+           {:method     "wakuext_collapsedCommunityCategories"
+            :params     []
+            :on-success [:communities/fetched-collapsed-categories-success]
+            :on-error   #(log/error "failed to fetch collapsed community categories" %)}]]]}))
+
 (rf/reg-event-fx :community/fetch
  (fn [_]
-   {:json-rpc/call [{:method     "wakuext_serializedCommunities"
-                     :params     []
-                     :on-success #(rf/dispatch [:community/fetch-success %])
-                     :on-error   #(log/error "failed to fetch communities" %)}
-                    {:method      "wakuext_checkAndDeletePendingRequestToJoinCommunity"
-                     :params      []
-                     :js-response true
-                     :on-success  #(rf/dispatch [:sanitize-messages-and-process-response %])
-                     :on-error    #(log/info "failed to fetch communities" %)}
-                    {:method     "wakuext_collapsedCommunityCategories"
-                     :params     []
-                     :on-success #(rf/dispatch [:communities/fetched-collapsed-categories-success %])
-                     :on-error   #(log/error "failed to fetch collapsed community categories" %)}]}))
+   {:fx [[:json-rpc/call
+          [{:method      "wakuext_serializedCommunities"
+            :params      []
+            :on-success  [:community/fetch-success]
+            :js-response true
+            :on-error    #(log/error "failed to fetch communities" %)}]]
+         [:dispatch-later [{:ms 1200 :dispatch [:community/fetch-low-priority]}]]]}))
```

Comment:
Why do we choose 1200ms?

Reply:
I better add a quick comment here. The idea is to not dispatch exactly at the same time we are delaying other events, like 1000s is already picked up, so I preferred to put it a bit above to keep the queue free for more important stuff. These two low priority RPC calls could wait much longer I think, but I preferred to be conservative.

This is all just a special measure because at login we have way more events being processed, but delaying events with arbitrary amounts is obviously of limited power. It's a mitigation for login.